test/interceptors/csrf-token.test.js

```diff
@@ -85,4 +85,18 @@ describe('CSRF token', () => {
         expect(axiosMock.defaults.headers.requesttoken).toBe('123')
         expect(response?.status).toBe(200)
     })
+
+    it('intercepts a cancellation error', () => {
+        const cancelError = {
+            code: 'ERR_CANCELED',
+            message: 'canceled',
+            name: 'CanceledError',
+            stack: '',
+        }
+        interceptor(cancelError).catch(error => {
+            Object.keys(cancelError).forEach(key => {
+                expect(error[key]).toBe(cancelError[key])
+            })
```

Comment:
if you want to do a deep comparison you can use `expect(error).toEqual(cancelError))`

Ref https://jestjs.io/docs/expect#toequalvalue

Reply:
:+1: